databuilder/README.md

```diff
@@ -468,6 +468,39 @@ job = DefaultJob(
 job.launch()
 ```
 
+#### [DbtExtractor](https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/extractor/dbt_extractor.py "SQLAlchemyExtractor")
+This extractor utilizes the [dbt](https://www.getdbt.com/ "dbt") output files `catalog.json` and `manifest.json` to extract metadata and ingest it into Amundsen. The `catalog.json` and `manifest.json` can both be generated by running `dbt docs generate` in your dbt project. Visit the [dbt artifacts page](https://docs.getdbt.com/reference/artifacts/dbt-artifacts "dbt artifacts") for more information.
+
+The `DbtExtractor` can currently create the following:
+
+- Tables and their definitions
+- Columns and their definitions
+- Table level lineage
+- dbt tags (as Amundsen badges or tags)
+- Table Sources (e.g. link to GitHib where the dbt template resides)
+
+```python
+job_config = ConfigFactory.from_dict({
+    # Required args
+    f'extractor.dbt.{DbtExtractor.DATABASE_NAME}': 'snowflake',
```

Comment:
if we use dbt for extraction, should the database be called dbt?

Reply:
I think the database should be named the same database that Amundsen is also extracting metadata from so that metadata will be attached to existing tables / columns.

To give a practical example, if we use the Snowflake extractor then the value for the database is aptly called `snowflake`. The key for a given table would be something akin to `snowflake://cluster.schema/table`

In this use-case if we want to use dbt to help fill in the additional metadata and to populate table descriptions, lineage, etc. we will need to create the same key. If we pass in the value "dbt" here would yield something like `dbt://cluster.schema/table`

Reply:
make sense, thanks for the info.